### ISSUE TITLE
Affordances for rate limiting posts and minimum eth balance

### ISSUE THREAD

**Issue by sachben91** [hicommonwealth/commonwealth#8473]
### Description

Based on the decision in #8268 , we have decided to rate limit number of posts that a user can make within a contest, along with requiring that every participant in a contest has a non dust value of eth (> 0.0005).  This ticket addresses the user affordances related to this change. 

### User Stories / Acceptance Criteria

- When a user attempts to submit a third post to an active contest within a topic, the "create thread" button should be disabled and the reason communicated to the user as shown below 
<img width="558" alt="Screenshot 2024-07-15 at 4 50 15 PM" src="https://github.com/user-attachments/assets/613e5c71-20cd-490d-ab64-7de3377e3735">

- If a topic has more than one contest, then the user should be shown which contests they are eligible to submit entries to and which ones have reached the rate limit of two posts per contest 
<img width="507" alt="Screenshot 2024-07-15 at 4 53 52 PM" src="https://github.com/user-attachments/assets/f2043868-77cb-44e9-812a-36831c1fb372">

- If the wallet that the user has connected has less than the non dust value of eth required, they should be shown the following message below the text box
<img width="486" alt="Screenshot 2024-07-16 at 12 29 56 PM" src="https://github.com/user-attachments/assets/29536316-887a-4c56-882c-2c5997c25bbe">

### Design Devlink

[designs ](https://www.figma.com/design/NNqlhNPHvn0O96TCBIi6WU/Contests?node-id=1887-21163&t=rh19Cun2HtOH1IZV-0)

**Comment by sachben91** [comment]
related ticket https://github.com/hicommonwealth/commonwealth/issues/8466

**Comment by timolegros** [comment]
@sachben91 just checking in on the affordances for admins creating contests and potentially crippling their communities by selecting the wrong topics. Also shouldn't the copy for the token warning say "with at least 0.0005 **ETH**" rather than "tokens"?

**Comment by sachben91** [comment]
@timolegros creating a separate ticket for the admin affordance and yes thanks for the catch, will change that to 0.0005 eth

**Comment by sachben91** [comment]
Related ticket https://github.com/hicommonwealth/commonwealth/issues/8474

**Comment by masvelio** [comment]
@sachben91 
1. The [designs](https://www.figma.com/design/NNqlhNPHvn0O96TCBIi6WU/Contests?node-id=1887-21163&t=rh19Cun2HtOH1IZV-0) are based on the updated `new thread page` while we still have old `new thread page`. The main difference is that we want to show `Insufficient balance` error below the user address dropdown, however, we don't have possibility to select the address currently in this view. 

In this case we should either :
- allow user to click `create` button but then throw the toast error
- disable `create` button and place the insufficient balance "inline error" somewhere in the form 

Below is comparison between new vs current design for more context:

NEW DESIGN
![image](https://github.com/user-attachments/assets/4e508768-af5e-4c5a-b336-ddf0303b73d9)

CURRENT DESIGN
![image](https://github.com/user-attachments/assets/32e280bd-6d42-4ac2-8d99-9e5b85f67950)

2. For those two points:
> When a user attempts to submit a third post to an active contest within a topic, the "create thread" button should be disabled and the reason communicated to the user as shown below

and 

> If a topic has more than one contest, then the user should be shown which contests they are eligible to submit entries to and which ones have reached the rate limit of two posts per contest

we need a backend logic from @rbennettcw. Basically, when we load `new thread page` we need to be able to display the data right away. Ideally for FE, we should extend `GET /topics` endpoint. We would need structure like this:
```
  topics: {
    //existing
    topicId: number;
    topicName: string;
    // added (only active contests)
    contests: { 
      contestId: string;
      contestAddress: string;
      contestName: string;
      submittedEntries: number;
    }[];
  }[];
  ```

TLDR:
@sachben91 => let me know whats the decision on point 1 
@rbennettcw => let us know if point numer 2 works for you

**Comment by sachben91** [comment]
@masvelio @jessmart1213 

>  disable create button and place the insufficient balance "inline error" somewhere in the form
I think this would be ideal solution because the information is really important. Perhaps the inline error can be under the "topic has ongoing contest" banner and to the left of create thread - Leaving this to jess

**Comment by sachben91** [comment]
@masvelio ticket has been updated
